src/sdl2_compat.c

```diff
@@ -3377,10 +3377,7 @@ SDL_RenderTargetSupported(SDL_Renderer *renderer)
     if (ret == 0) {
         /* SDL_RENDERER_TARGETTEXTURE was removed in SDL3, check by name for 
          * renderer that does not support render target. */
-        if (SDL3_strcmp(info.name, "opengles") == 0) {
-            return SDL_FALSE;
-        }
-        return SDL_TRUE;
+        return !!SDL3_strcmp(info.name, "opengles");
```

Comment:
This probably shouldn't be changed, for clarity.

Reply:
Done